src/plots/plot.ts

```diff
@@ -452,6 +455,13 @@ export class Plot extends Component {
     this._additionalPaint(maxTime);
   }
 
+  protected _deferredTotalDrawTime() {
+    let drawSteps = this._paintedDrawSteps;
```

Comment:
The problem is that we need the draw steps in order to determine the time it takes to draw.  However, once a plot calls `_rendersImmediately()` then we cannot retrieve the time it takes to draw (Look at the `_getAnimator()` endpoint for more details).

This is one specific part that I feel a bit iffy about but I couldn't think of an alternative solution.